src/Skeletonizer/SkeletonImage.ts

```diff
@@ -1,397 +1,386 @@
 "use strict";
```

Comment:
Remove all "use strict"  in all files: this is by default when using ESM syntax.